.github/workflows/kfp-kubernetes-test.yaml

```
@@ -0,0 +1,37 @@
+name: KFP K8s Platform Tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    paths:
+      - '.github/workflows/kfp-kubernetes-library.yaml'
```

This should match the name of this file. Like:

```suggestion
      - '.github/workflows/kfp-kubernetes-test.yaml'
```

---

Done